src/smokeTest/java/uk/gov/hmcts/reform/bulkscan/orchestrator/JurisdictionAuthenticationTest.java

```diff
@@ -0,0 +1,36 @@
+package uk.gov.hmcts.reform.bulkscan.orchestrator;
+
+import com.typesafe.config.ConfigFactory;
+import io.restassured.RestAssured;
+import io.restassured.mapper.TypeRef;
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+import org.springframework.http.HttpStatus;
+import uk.gov.hmcts.reform.logging.appinsights.SyntheticHeaders;
+
+import java.util.Map;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+public class JurisdictionAuthenticationTest {
+
+    private static final String TEST_URL;
+
+    static {
+        TEST_URL = ConfigFactory.load().getString("test-url");
+    }
+
+    @DisplayName("Each configured jurisdiction should have valid credentials")
+    @Test
+    public void jurisdictions() {
+        Map<String, HttpStatus> response = RestAssured
+            .given()
+            .relaxedHTTPSValidation()
+            .baseUri(TEST_URL)
+            .header(SyntheticHeaders.SYNTHETIC_TEST_SOURCE, "Bulk Scan Orchestrator smoke test")
+            .get("/jurisdictions")
+            .as(new TypeRef<Map<String, HttpStatus>>() {});
+
+        assertThat(response.values()).containsOnly(HttpStatus.OK);
```

Comment:
Maybe we should compare the whole maps, so that when there's a problem with one of the jurisdictions, the test will tell us which one.